modules/rosa-hcp-sgs-and-vpce.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sgs-and-vpce_{context}"]
+= Configuring AWS security groups and privatelink connections
+
+After creating your VPC, create your AWS security groups and VPC endpoints.
+
+.Procedure
+
+. Create the AWS security group by running the following command:
++
+[source,terminal]
+----
+$ aws ec2 create-security-group \
+        --group-name allow-inbound-traffic 
+        --description "allow inbound traffic" 
+        --vpc-id $VPC_ID
+----
+
+. Grant access to the security group's ingress by running the following command:
++
+[source,terminal]
+----
+$ aws ec2 authorize-security-group-ingress \
+        --group-id $SECURITY_GROUP_ID \
+        --protocol -1 \
+        --port 0-0 \
+        --cidr <vpc_cidr> \ <1>
+----
+<1> Enter the CIDR of your VPC.
+
+. Create your STS VPC endpoint by running the following command:
++
+[source,terminal]
+----
+$ aws ec2 create-vpc-endpoint \
+    --vpc-id $VPC_ID \
+    --service-name com.amazonaws.${REGION}.sts \
+    --vpc-endpoint-type Interface
+----
+
+. Create your ECR VPC endpoints by running the following command:
++
+[source,terminal]
+----
+$ aws ec2 create-vpc-endpoint \
+    --vpc-id $VPC_ID \
+    --service-name com.amazonaws.${REGION}.ecr.api \
+    --vpc-endpoint-type Interface  
+----
++
+[source,terminal]
+----
+$ aws ec2 create-vpc-endpoint \
+    --vpc-id $VPC_ID \
+    --service-name com.amazonaws.${REGION}.ecr.dkr \ <1>
+    --vpc-endpoint-type Interface  
+----
+<1> Enter your AWS region where the VPC is located.
+
+. Create your S3 VPC endpoint by running the following command:
++
+[source,terminal]
+----
+$ aws ec2 create-vpc-endpoint \
+    --vpc-id $VPC_ID \
+    --service-name com.amazonaws.${REGION}.s3 
```

Comment:
See my previous comment. I suspect that all of the instances of `${REGION}.s3` and `$VPC_ID` should be `<region>` or `<vpc_id>`, respectively. Again, if this is not the case and the customer should enter these exact values due to the script mentioned in the first module of the assembly or some sort of environment variable, I think that should be called out.